lib/order_processor.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'list_of_dishes'
+require_relative 'send_sms'
+
+class OrderProcessor
+
+  attr_reader :list_of_dishes, :orders, :number_of_orders, :overall_price, :sms_message_sender
+
+  def initialize(list_of_dishes = ListOfDishes.new)
+    @list_of_dishes = list_of_dishes
+    @orders = {}
+    @number_of_orders = 0
+    @overall_price = 0
+    @sms_message_sender = SMSMessage.new
+  end
+
+  def add_order(order_number, quantity)
+    if @orders.key?(order_number.to_s.to_sym)
```

Comment:
Is it necessary for you to go to string and then to symbol? Would it not be able to convert straight to a symbol?